internal/cmd/operator-sdk/run/packagemanifests/packagemanifests.go

```diff
@@ -27,7 +27,10 @@ import (
 func NewCmd(cfg *operator.Configuration) *cobra.Command {
 	i := packagemanifests.NewInstall(cfg)
 	cmd := &cobra.Command{
-		Use:   "packagemanifests [packagemanifests-root-dir]",
+		Use: "packagemanifests [packagemanifests-root-dir]",
+		Deprecated: "packagemanifests format is deprecated and will soon be removed. Use bundles to " +
```

Comment:
```suggestion
		Deprecated: "support for the packagemanifests format will be removed in operator-sdk v2.0.0. Use bundles to " +
```

Reply:
Can you make this deprecation message the same as that for `generate packagemanifests`?